src/std/fs.ab

```diff
@@ -1,55 +1,66 @@
+import * from "std/env"
+
+/// Check if directory exists
 pub fun dir_exist(path) {
     $[ -d "{path}" ]$ failed {
         return false
     }
     return true
 }
 
+/// Check if file exists
 pub fun file_exist(path) {
     $[ -f "{path}" ]$ failed {
         return false
     }
     return true
 }
 
+/// Get the file content
 pub fun file_read(path) {
     return $< "{path}"$?
 }
 
+/// Write the content to the file, doesn't check if the file exist
 pub fun file_write(path, content) {
     return $echo "{content}" > "{path}"$?
 }
 
+/// Append the content to the file, doesn't check if the file exist
 pub fun file_append(path, content) {
     return $echo "{content}" >> "{path}"$?
 }
 
+/// Create a symbolic link, if the file doens't exist return a boolean and print a message
 pub fun create_symbolic_link(origin: Text, destination: Text): Bool {
     if file_exist(origin) {
         unsafe $ln -s "{origin}" "{destination}"$
         return true
     }
 
-    echo "The file {origin} doesn't exist!"
+    error("The file {origin} doesn't exist!")
     return false
 }
 
+/// Create a directory with the full path
 pub fun create_dir(path: Text): Null {
     if not dir_exist(path) {
         unsafe $mkdir -p "{path}"$
     }
 }
 
+/// Set the file as executable, if the file doesn't exist return a boolean and print a message
 pub fun make_executable(path: Text): Bool {
     if file_exist(path) {
         unsafe $chmod +x "{path}"$
         return true
     }
 
-    echo "The file {path} doesn't exist!"
+    error("The file {path} doesn't exist!")
```

Comment:
We can use the new function to color the output

Reply:
also just fail